style.css

```diff
@@ -116,3 +116,365 @@
   gap: 19px;
   align-items: baseline;
 }
+
+/* #work-container {
+  display: grid;
+  flex-direction: column;
+  gap: 20px;
+  padding-top: 12px;
+  background-color: #fff;
+} */
+
+#work-container {
+  display: grid;
+  flex-direction: column;
+  gap: 20px;
+  padding-top: 12px;
+  background-color: rgba(0, 0, 0, 0.1);
+}
+
+/* .container {
+  display: grid;
+  grid-template-columns: auto;
+  gap: 10px;
+  background-color: #fff;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+  margin: 50px auto;
+  padding: 16px 0;
+  position: static;
+} */
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** Kindly make sure to delete all the commented code lines to make your code base cleaner and professional.

Reply:
@tresorsawasawa Noted! Thank you.